hazelcast/src/main/java/com/hazelcast/sql/impl/parser/SqlParser.java

```diff
@@ -14,19 +14,18 @@
  * limitations under the License.
  */
 
-package com.hazelcast.sql.impl.optimizer;
-
-import com.hazelcast.sql.impl.plan.Plan;
+package com.hazelcast.sql.impl.parser;
 
 /**
- * Optimizer responsible for conversion of SQL string to executable plan.
+ * Parser responsible for conversion of SQL string to executable statement.
  */
-public interface SqlOptimizer {
+public interface SqlParser {
```

Comment:
As mentioned before, the motivation of the rename is not obvious. This underlying implementation is optimizer, not parser.